hugegraph-dist/pom.xml

```diff
@@ -215,4 +193,40 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>unix-download-swagger-ui</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                    </plugin>
+                </plugins>
+            </build>
```

Comment:
can extract to root pom file?

Reply:
@z7658329 done in [`509a4e2` (#2277)](https://github.com/apache/incubator-hugegraph/pull/2277/commits/509a4e27f8f58807be4c36019b7eb1330b01d63a)

Reply:
> can extract to root pom file?

better to keep them in dist module now